languages/translations/ja.json

```diff
@@ -299,90 +299,90 @@
         },
         "off ramp": {
             "default": {
-                "default": "ランプに向かう",
-                "name": "{way_name}に向けてランプに向かう",
-                "destination": "{destination}に向けてランプに向かう",
-                "exit": "{exit}出口に向かう",
-                "exit_destination": "{exit}出口を出て{destination}へ向かう"
+                "default": "出口です",
+                "name": "{way_name}方面、出口です",
```

Comment:
"{way_name}方面、出口に向かう" is better
"{way_name}方面、出口です" means "there is an exit on the {way_name}"

Reply:
I would probably just replace 出口です with 出口に向かう

Reply:
I believe many of Japanese Navigation system will say "出口です” followed with "右側車線を進んでください"(take right lane).  @danpaz we may need to get more user research if you are uncertain for that.

Reply:
@tichimura that is good to know about combining 出口です with lane instructions. This project doesn't support lanes yet, so we could not distinguish between a road with one lane or multiple. At this time it sounds like @tsuz's suggestion to replace all instances of 出口です with 出口に向かう would be the next best option. Do you agree with that?

Reply:
@danpaz 

@tichimura has a great point since he has done more extensive comparison with other car navigations. "出口です" seems to be standard among other navs whereas my suggestion was to keep the format standardized among other strings. So, whichever is fine!

Reply:
👍 thanks both, I kept the original phrasing for off ramp (出口です) and on ramp (入口です).